### DISCUSSION TITLE
Add / customize data inspector question

### DISCUSSION THREAD

**Discussion by mhite** [WerWolv/ImHex#754] [Get Help] [answered]
Is it possible to customize the data inspector view? I would like to show utf-16BE strings when I highlight in the hex editor view.

**Comment by WerWolv** [comment, marked as answer]
Hey
There's currently no way to add a new row to the data inspector unless you write a plugin for ImHex.
That said, the latest commit adds a Wide String row which should work for your usecase. Standard is UTF-16LE but you can change the endian switch to "Big" for it to decode UTF-16BE instead.

Commit: https://github.com/WerWolv/ImHex/commit/b450f4797ee6800e2688b0175b8944c5bcd167e2

**Reply by mhite** [reply]
Excellent, thanks! I will look for it in the next release!
